jobs/cam/stats/JEVS_CAM_STATS

```diff
@@ -1,69 +1,74 @@
+
 export pid=$$
-# in production, the following will be set to DATAROOT
-#export DATA_IN=${DATA_IN:-$DATAROOT}
-export DATA_IN=${DATA_IN:-$DATAROOTtmp}
-export DATA=$DATA_IN/${jobid}
-rm -rf $DATA
+
+export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
 mkdir -p $DATA
 cd $DATA
-
-# General I/O Settings
+####################################
+# Determine Job Output Name on System
+####################################
 export pgmout="OUTPUT.${pid}"
-# in production the following will be set to yesterday's date
-export VDATE=$(date -d "today -1 day" +"%Y%m%d")
-export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
-# in production the following will be set using compath.py
-#export COMINccpa=${COMINccpa:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT/$RUN)}
-export COMINccpa=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT/$RUN
-# in production the following will be set using compath.py
-#export COMINmrms=${COMINmrms:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT/$RUN)}
-export COMINmrms=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT/$RUN
-export COMINnam=${COMINnam:-$(compath.py ${envir}/com/nam/${nam_ver})}
-export COMINhiresw=${COMINhiresw:-$(compath.py ${envir}/com/hiresw/${hiresw_ver})}
-export COMINhrrr=${COMINhrrr:-$(compath.py ${envir}/com/hrrr/${hrrr_ver})}
-# in production, the following will be set to the approp. COMOUT dir
-#export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver/$STEP/$COMPONENT/$RUN)}
-export COMOUT=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/$STEP/$COMPONENT/$RUN
 
-# Subdirectories to EVS Home Directory
-export PARMevs=$HOMEevs/parm
-export USHevs=$HOMEevs/ush
-export EXECevs=$HOMEevs/exec
-export FIXevs=$HOMEevs/fix
-export SCRIPTSevs=$HOMEevs/scripts
-export MET_PLUS_CONF="${PARMevs}/metplus_config/cam/precip/stats"
+export cycle=t${cyc}z
+##################################################
+# SAVEGES  - Copy Files From TMPDIR to $GESdir
+# SENDECF  - Flag Events on ecFLOW
+# SENDCOM  - Copy Files From TMPDIR to $COMOUT
+# SENDDBN  - Issue DBNet Client Calls
+##################################################
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-YES}
+export SENDECF=${SENDECF:-YES}
+
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${model_ver}}
+export EXECevs=${EXECevs:-$HOMEevs/exec}
+export FIXvevs=${FIXevs:-$HOMEevs/fix}
+export PARMevs=${PARMevs:-$HOMEevs/parm}
+export USHevs=${USHevs:-$HOMEevs/ush}
+
+# Define COMIN/COMOUT variables:
 
-# Execute the Script
-$SCRIPTSevs/$COMPONENT/$STEP/exevs_${MODELNAME}_${VERIF_CASE}_${STEP}.sh
-export err=$?; err_chk
+export COMIN=${COMIN:-$(compath.py ${envir}/com/evs/${evs_ver}/)}
+export COMINnam=${COMINnam:-$(compath.py ${envir}/com/${modsys}/${mod_ver})}
+export COMINobs=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
+export COMOUT=${COMOUT:-$(compath.py ${NET}/${evs_ver}/${STEP}/${COMPONENT}/${RUN})}
```

Comment:
I suggest we add a failsafe line after the COMOUT definition in _**all J-job scripts across EVS**_:
`mkdir -m 775 -p $COMOUT`
This will assure that COMOUT exists and will not cause downstream processing to try to write to a directory that doesn't exist. This line is in all operational J-job scripts if you care to poke around in $PACKAGEROOT for more context. 
My initial tests weren't working b/c I defined COMOUT as a path that didn't yet exist. So, to stop that problem in the future, this line says "create this directory if it doesn't yet exist and give it writable permissions."